ca/server.go

```diff
@@ -60,7 +62,7 @@ type Server struct {
 	// the security config as a result
 	lastSeenClusterRootCA *api.RootCA
 	lastSeenExternalCAs   []*api.ExternalCA
-	secConfigMu           sync.Mutex
+	signingMu             sync.Mutex
```

Comment:
I guess it's supposed to protect the components of the CA server used to issue new certificates, so the rootCA object, and everything needed to connect to an external CA (hence the parts of `securityConfig`, because it has the TLS credentials needed to connect to the external CA)